lib/rbs/unit_test/spy.rb

```diff
@@ -128,7 +128,7 @@ def wrapped_object
                 end
               end.ruby2_keywords
             )
-          end.new()
+          end.allocate()
```

Comment:
This is needed because `.new` implicitly calls `.initialize`; if `method_name` is `:initialize`, then this will accidentally call it with no arguments. Doing it this way prevents that (and since the parent object is `BasicObject`, `:initialize` is a no-op anyways)

Reply:
BTW, why not [`MiniTest::Mock`](https://ruby-doc.org/current/gems/minitest/Minitest/Mock.html)?